### PR TITLE
Changed the way how requests to the GitHub API were authenticated

### PR DESCRIPTION
**Pull request by Andarist**
fixes https://github.com/changesets/action/issues/51

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "typescript.tsdk": "node_modules/typescript/lib"
```
Comment (mitchellhamilton) [inline .vscode/settings.json:2]: Why this change?

**Comment by calebeby** [comment]
Thank you!

File: .vscode/settings.json (hunk 1 above)
Comment (Andarist) [reply .vscode/settings.json:2]: Oh, I wasn't supposed to add this to this commit - it's a local change that I'm using. I had to accidentally add it.

It is good though - by default VSCode is using its builtin TS version and this often mismatches with the one actually used by the project itself. This makes it to use a local version - which should always be preferred.
